_sections/videocalling-android-studio.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Visual Studio Code
+tab: vscode
+---
+
+## Overview
+
+GitLive's video-calling feature allows you to video call teammates straight from your IDE and work together on files exactly like you would in a Google Doc. This feature will undoubtedly boost your productivity when you want to fix code issues with a teammate.
```

Comment:
GitLive's video-calling feature allows you to call teammates straight from your IDE and work together on files like you would on a Google Docs collaboration. This feature will undoubtedly boost your productivity when you want to fix code issues with a teammate.